journal/modernized/26-chapter-23.adoc

```diff
@@ -140,13 +141,13 @@ having long to subsist on fish and cocoa-nuts.
 He appeared glad to see me; and after sitting awhile by the bedside,
 I began to advert to his appalling situation, winding gradually on as matter opened.
 On saying it was little matter how soon we leave this world of trouble,
-if we are but prepared for the event; he said, '`I am not prepared,
-and cannot prepare myself.`' I told him I was even rejoiced to
+if we are but prepared for the event; he said, "`I am not prepared,
+and cannot prepare myself.`" I told him I was even rejoiced to
 find he was thus sensible of his own inability and weakness,
 because it was a conscious feeling of the necessity of the Saviour`'s help.
 I endeavoured to turn his mind to the dear Redeemer; but he said,
-'`the time was too short to expect to accomplish the great work,
-'` and spoke as if it had been too long deferred.
+"`the time was too short to expect to accomplish the great work,
+'" and spoke as if it had been too long deferred.
```

Comment:
Let's remove the `'"` on this line, and move it up to the line before, closing the quotation there, like so:

```ASCIIDOC
"`the time was too short to expect to accomplish the great work,`"
```

Reply:
I had wondered about that but assumed it was okay because it looked okay in the book.